### ISSUE TITLE
Error: convert needs fontconfig

### ISSUE THREAD

**Issue by samtgarson** [vvo/gifify#29]
Hey there.

I'm trying to convert using the following command:
```
gifify tide\ 1_.mov -o tide_backend.gif --resize '900:-1' 
```
however I get the following error without anything else:
```
convert needs fontconfig
```

Any help please? Cheers

**Comment by wesbos** [comment]
I got this too - even when I had all the dependencies installed.

I just opened the file and removed the need for these deps because I don't need captions

https://github.com/vvo/gifify/blob/master/bin/gifify#L92-L93

**Comment by samtgarson** [comment]
Works for me, thanks.

**Comment by joemccann** [comment]
Still an issue without @wesbos's change, right?

**Comment by wesbos** [comment]
Yeah - this should be changed in the source. It should only check for `libass` and `fontconfig` if the thing we are doing requires those things - basic mp4→gif doesn't need them

**Comment by vvo** [comment]
Open a PR, I will merge

**Comment by vvo** [comment]
cc @joemccann @wesbos

**Comment by ispal** [comment]
I had this issue, but in my case I had MAMP on my machine which included Imagemagick with different settings. It defaulted to MAMP's version of the library without fontconfig because I had MAMP's bin folder in $PATH before /usr/local in .zshrc. So, check your settings if you are experiencing this issue.

**Comment by timarney** [comment]
MAMP was my issue as well.

**Comment by skovmand** [comment]
Also got this problem. Frustrating.

**Comment by vvo** [comment]
One way to solve this @skovmand would be to be not that picky about requirements and let error propagates when the user actually tries to use such flag like captions: https://github.com/vvo/gifify/blob/master/bin/gifify#L92-L93

Open to a PR solving this all

**Comment by ahmadawais** [comment]
Simply uninstall all the dependencies and reinstall them as stated in the readme.md file :)

**Comment by sitexa** [comment]
Same problem encounter today.

**Comment by maxp-edcast** [comment]
uninnstallinng imagemagi ck and reinstallingn with the flag did not fix this issue.

**Comment by kahlilashanti** [comment]
installing [per the documentation](https://github.com/vvo/gifify#requirements) with flags did not solve it for me.  any updates?  thanks!

**Comment by ericat** [comment]
Same, I get:
```
Warning: ffmpeg: this formula has no --with-fontconfigg option so it will be ignored!
```
